gradle/java.gradle

```diff
@@ -12,6 +12,7 @@ compileJava {
 
 task testJar(type: Jar, dependsOn: testClasses, group: "build") {
     appendix = 'test'
+    classifier = 'tests'
```

Comment:
That is okay, because this doesn't change it. It only affects the name for deployment